src/index.js

```diff
@@ -287,7 +305,11 @@ class ObservationsVisualizer extends GenericVisualizer {
         { title: 'LOINC', versions: '*', getter: o => o.code.coding[0].code },
         { title: 'Observation', versions: '*', getter: o => o.code.coding[0].display },
         { title: 'Value', versions: '*', getter: o => obsValue(o) },
-        { title: 'Date Recorded', 'versions': '*', format: 'date', getter: o => o.effectiveDateTime }
+        { title: 'Effective Date', 'versions': '*', format: 'date', getter: o => o.effectiveDateTime },
```

Comment:
In cases like this where there's `effective[x]` and we know only one of the types can exist at a time, would it maybe make more sense to have just one column and it chooses the value that's present? Ex:

```js

function getEffective(obs) {
  if (obs.effectiveDateTime) return obs.effectiveDateTime;
  if (obs.effectivePeriod) {
    return `${obs.effectivePeriod.start} - ${obs.effectivePeriod.end}`;
  }
  // other effective[x]
}

...

{ title: 'Effective', getter: o => getEffective(o) }
```

Reply:
I added a function `attributeXTime` that's hopefully even a bit more flexible so that it can work with effective[x] or performed[x] or any of those where we may want to handle either a period or datetime for the same essential field. Lemme know what you think!